doc/docs/master/concepts/pipeline-concepts/datum/index.md

```diff
@@ -1,12 +1,41 @@
 # Datum
```

Comment:
not changes suggested in 1.9.x version and apply here.

Reply:
added